nginx.conf

```diff
@@ -42,6 +42,55 @@ server {
         add_header Cache-Control "public";
     }
 
+    location ${BASE_PATH}/android-chrome-192x192.png {
```

Comment:
Let's move all these files into a folder: `public/manifest`
Then the nginx location can just be:
```
location ${BASE_PATH}/manifest/ {
    alias /usr/share/nginx/html/icons/;
    expires 1y;
    access_log off;
    add_header Cache-Control "public";
}
```

Reply:
nice catch, here's the fix: [e807301](https://github.com/powerhouse-inc/connect/pull/532/commits/e80730199bd9401f6823c30944d305bd63d0a854)